packages/xo-web/src/common/utils.js

```diff
@@ -646,10 +646,14 @@ export const createCompare = criterias => (...items) => {
 
 // ===================================================================
 
-export const hasLicenseRestrictions = host =>
-  host.productBrand !== 'XCP-ng' &&
-  versionSatisfies(host.version, '>=7.3.0') &&
-  host.license_params.sku_type === 'free'
+export const hasLicenseRestrictions = host => {
+  const licenseType = host.license_params.sku_type
+  return (
+    host.productBrand !== 'XCP-ng' &&
+    versionSatisfies(host.version, '>=7.3.0') &&
+    (licenseType === 'free' || licenseType === 'express')
+  )
+}
```

Comment:
The ideal configurable code should like this
```js
const licenseRestrictionsPerProduct = {
  xenServer: {
    restrictionLicences: ["free", "express"],
    versionRestriction: ">=7.3.0"
  },
  "XCP-ng": {
    restrictionLicences: [],
    restrictionMinVersion: "0.0.0"
  }
};

export const hasLicenseRestrictions = host => {
  const restrictions = licenseRestrictionsPerProduct[host.productBrand];
  return (
    restrictions &&
    versionSatisfies(host.version, restrictions.versionRestriction) &&
    restrictions.restrictionLicences.includes(host.license_params.sku_type)
  );
};
```

But since we have only two brands (xcp-ng, xenServer) I'm okey with your solution.

Reply:
Also, since 8.0, `XenServer` became `Citrix Hypervisor` so you'd need a third case. So yes, let's keep it like that, it's good enough :)